File: php/#2/vezba.php
<?php

// 1. Napisati niz (array) sa 3 clana (Bmw, Mercedes, Audi)
// 2. Dodati 1 novog clana u niz koji se zove Zastava
// 3. Ispisati clanove niza u sledecem redosledu Zastava, Mercedes, Audi, Bmw

$automobili = ["Bmw", "Mercedes", "Audi"];
array_push($automobili, "Zastava");

echo "$automobili[3] $automobili[1] $automobili[2] $automobili[0]";

$brojAutomobila = count($automobili); // 4, $brojAutomobila = 4

echo "Ukupno imamo $brojAutomobila automobila";

var_dump($automobili);

$ime = "Toma";
var_dump($ime);

File: python/2/vezba.py

#         0       1        2
cars = ["Audi", "BMW", "Zastava"]

print(cars)

# BMW -> Mercedes
cars[1] = "Mercedes"
print(cars)

# Dodati jos 1 automobil na kraj "Skoda"
cars.append("Skoda")
print(cars)

# Sortirajte cars A-Z
cars.sort()
print(cars)

# Trenutno na stanju imamo X automobila
print(f"Trenutno na stanju imamo {len(cars)} automobila")